pandas/tests/io/excel/test_readers.py

```diff
@@ -943,11 +943,18 @@ def test_reader_seconds(self, request, engine, read_ext):
                 ]
             }
         )
-
-        actual = pd.read_excel("times_1900" + read_ext, sheet_name="Sheet1")
+        if engine == "odf":
+            # odf recognises cell type as time (from its attribute)
```

Comment:
Is this an issue with the file itself? Wondering if this needs to be fixed for this test and then created as a separate test for the odf specific behavior

Reply:
sure, broken off into https://github.com/pandas-dev/pandas/pull/49712

Reply:
fixed, no need to modify this test file any more in this PR